ui/src/app/apps/app-details/app-details.component.html

```diff
@@ -0,0 +1,53 @@
+<h1 [ngBusy]="busy">Application Details</h1>
+
+<table class="table table-striped table-hover table-fixed" *ngIf="detailedAppRegistration">
+  <tbody>
+    <tr>
+      <th>
+        Name
+      </th>
+      <td>
+        {{detailedAppRegistration.name}}
+      </td>
+    </tr>
+    <tr>
+      <th>
+        Type
+      </th>
+      <td>
+        {{detailedAppRegistration.type}}
+      </td>
+    </tr>
+    <tr>
+      <th>
+        Uri
+      </th>
+      <td>
+        {{detailedAppRegistration.uri}}
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<h2>Application Properties</h2>
```

Comment:
May consider not showing this header and skip showing the "no properties" message if there is no detailedAppRegistration.  (i.e `*ngIf="detailedAppRegistration"`).  Because there will be no TBody above.  The ToastyService error enumerated below will tell them why.